src/librbd/operation/DisableFeaturesRequest.cc

```diff
@@ -190,7 +192,8 @@ Context *DisableFeaturesRequest<I>::handle_acquire_exclusive_lock(int *result) {
     }
     if ((m_features & RBD_FEATURE_OBJECT_MAP) != 0) {
       if ((m_new_features & RBD_FEATURE_FAST_DIFF) != 0) {
-        lderr(cct) << "cannot disable object map" << dendl;
+        lderr(cct) << "cannot disable object-map. fast-diff must be "
+                      "disabled before disabling object-map" << dendl;
```

Comment:
Nit: inconsistent use of periods -- add a period to the end of the last sentence here and below where needed.